**Individual planning report** by **Cavas Ma**



**[1] Data Description: players.csv**

In [13]:
library(tidyverse)

In [12]:

player_url <- "https://raw.githubusercontent.com/ca-vas/DSCI-100-project/refs/heads/main/players.csv?token=GHSAT0AAAAAADPKBNJEALNZ324HFLIY7YKG2IYI6CA"

players <- read_csv(player_url)

summary(players)

exp_lvls <- table(players$experience)
print(exp_lvls)


Rows: 196 Columns: 7
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (4): experience, hashedEmail, name, gender
dbl (2): played_hours, Age
lgl (1): subscribe

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


  experience        subscribe       hashedEmail         played_hours    
 Length:196         Mode :logical   Length:196         Min.   :  0.000  
 Class :character   FALSE:52        Class :character   1st Qu.:  0.000  
 Mode  :character   TRUE :144       Mode  :character   Median :  0.100  
                                                       Mean   :  5.846  
                                                       3rd Qu.:  0.600  
                                                       Max.   :223.100  
                                                                        
     name              gender               Age       
 Length:196         Length:196         Min.   : 9.00  
 Class :character   Class :character   1st Qu.:17.00  
 Mode  :character   Mode  :character   Median :19.00  
                                       Mean   :21.14  
                                       3rd Qu.:22.75  
                                       Max.   :58.00  
                               


 Amateur Beginner      Pro  Regular  Veteran 
      63       35       14       36       48 


**Dataset summary:**

Number of rows: 196 <br>
Number of columns(variables):7 <br>
| Variables | Types of var| Meaning | Data collection method|
|-------|-----|---------|-----------|
| experience  | character  |  player's experience of playing minecraft, categorized into 5 levels: Amateur,Beginner,Pro, Regular, Veteran | self-report |
| subscribe | logical  | whether the player has subscribed to the Minecraft newsletter  |either self-report or linked to self-reported email|
| hashedEmail | character | user identification, hashed for privacy reasons   |self-reoprt, verified by player and server|
| played_hours | double | number of hours played on Minecraft |recorded on Minecraft server|
| name  | character | player's names |self-report|
| gender | character  | player's gender  |self-report|
| Age | double  | player's age  |self-report|


Mean playtime: 5.85hr <br>
Mean age: 21.14 yaers old <br>


Observable issue: self-reported data can be inaccurate or made up, affecting the overall correctness of the data.